testing/resources/python3_hooks_repo/build/lib/py3_hook.py

```diff
@@ -0,0 +1,8 @@
+import sys
```

Comment:
oops, [don't run `git add .`](https://youtu.be/GgIZbO2Hb5s)

Reply:
Shoot. I thought I'd fixed that.... Will do.